### PR TITLE
Install libcuda.so.1 to QEMU sysroot

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

cc @h-vetinari

### CODE REVIEW

File: recipe/cross_compile_support.sh [hunk 1]
```diff
@@ -108,6 +109,9 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
                 mkdir -p /opt/conda/targets/
                 mv ./usr/local/cuda-${CUDA_COMPILER_VERSION}/targets/${CUDA_HOST_PLATFORM_ARCH}-linux /opt/conda/targets/${CUDA_HOST_PLATFORM_ARCH}-linux
                 /usr/bin/sudo cp -r /opt/conda/targets/${CUDA_HOST_PLATFORM_ARCH}-linux ${CUDA_HOME}/targets/${CUDA_HOST_PLATFORM_ARCH}-linux
+
+		# Need libcuda.so.1 in test phase to test the executables
+		mv ./usr/local/cuda-${CUDA_COMPILER_VERSION}/compat/* ${QEMU_LD_PREFIX}/usr/lib/
```
Comment (beckermr) [inline recipe/cross_compile_support.sh:114]: bash may not care but it looks like this block needs to be indented maybe?

Comment (isuruf) [reply recipe/cross_compile_support.sh:114]: I'm not sure I understand. They have the same indentation as Line 110 right?

Comment (beckermr) [reply recipe/cross_compile_support.sh:114]: I don't think so.

Screenshot 2023-04-18 at 10 45 10 AM

Comment (beckermr) [reply recipe/cross_compile_support.sh:114]: Are there tabs not being rendered correctly?

Comment (isuruf) [reply recipe/cross_compile_support.sh:114]: This is what I see

image

Comment (beckermr) [reply recipe/cross_compile_support.sh]: OMG.

**Pull request merged**